README.md

```diff
@@ -2,4 +2,52 @@
 
 *⚠️ Note: This repository is a WIP, "watch" it to keep up with updates!*
 
-This repository hosts Bokeh equivalents for various plots from ["Fundamentals of Data Visualization" by Claus O. Wilke](https://clauswilke.com/dataviz/).
+## Project description
+
+This repository hosts Bokeh equivalents for various plots from ["Fundamentals of Data Visualization" by Claus O. Wilke](https://clauswilke.com/dataviz/). It provides a collection of interactive data visualizations implemented using the Bokeh library.
```

Comment:
Being *excessively* pedantic, but a book title is normally italicized rather than quoted
```suggestion
This repository hosts Bokeh equivalents for various plots from [*Fundamentals of Data Visualization* by Claus O. Wilke](https://clauswilke.com/dataviz/). It provides a collection of interactive data visualizations implemented using the Bokeh library.
```

Reply:
Done.